### PR TITLE
Solaris11 support

### PR DESCRIPTION
**Pull request by juliengk**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -1292,6 +1339,33 @@
           }
         }
         '5.11': {
+          file { 'pam_d_cron':
```
Comment (ghoneycutt) [inline manifests/init.pp]: Curious why these files were not already in the module since the module already supposedly supports Solaris 11

Comment (juliengk) [inline manifests/init.pp]: good question!

**Comment by ghoneycutt** [comment]
Looks like @kentjohansson originally added support for Solaris 11. Since I don't run that platform, we'll just need confirmation from another source/hub that these changes are OK. We want the default values for Solaris 11, not the Ericsson defaults.

**Comment by juliengk** [comment]
I will ask another hub to test it once I will be ready. Thanks

**Comment by Phil-Friderici** [comment]
This version was successfully tested by a co-worker, he wrote:
> I have tested the pam module and now works ok on Solaris 11.1 and fails on Solaris 11.0 – as expected.

Guess it's time to fix the spec tests now...

**Pull request closed without merging**